libs/wire-api/src/Wire/API/Routes/Public/Brig.hs

```diff
@@ -1177,7 +1219,7 @@ type SearchAPI =
              (SearchResult TeamContact)
     )
 
-type MLSAPI = LiftNamed (ZLocalUser :> "mls" :> MLSKeyPackageAPI)
+type MLSAPI = LiftNamed ("mls" :> MLSKeyPackageAPI)
```

Comment:
Doesn't this removal break the whole API?

And how is it related to this PR?

Reply:
I inlined them into `MLSKeyPackageAPI` directly. The problem is that the `callsFed` combinator requires the `MakesFederatedCall` api modifications to come before any arguments that show up in the `ServerT` (of which `ZLocalUser` is one).

Reply:
Thanks for the explanation! Somehow this went under radar when I was reviewing it.